apps/scan/frontend/src/utils/use_session_settings_manager.ts

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import {
+  VoterSettingsManagerContext,
+  useCurrentLanguage,
+  useLanguageControls,
+} from '@votingworks/ui';
+import { DefaultTheme, ThemeContext } from 'styled-components';
+import { LanguageCode } from '@votingworks/types';
+
+export interface SessionSettingsManagerProps {
+  resetVoterSettings: () => void;
+  cacheAndResetVoterSettings: () => void;
+  restoreVoterSessionsSettings: () => void;
```

Comment:
Another way to think about these that might be clearer:

```ts
startNewSession
pauseSession
resumeSession
```

Reply:
Agreed